docs/distributions/debian/installation.md

```diff
@@ -1,57 +1,59 @@
 # Introduction
 
-This guide shall help you in installing a Debian or Ubuntu based Linux distro, which is not available as a modified ISO with T2 support.
+This guide will help you in installing a Debian or Ubuntu-based Linux distro that does not have a modified ISO with T2 support.
 
-Some popular distros include :-
+Some popular distros include:
 
-1. [Linux Mint](https://linuxmint.com/)
-2. [Pop!_OS](https://pop.system76.com/)
-3. [elementary OS](https://elementary.io/)
-4. [Debian](https://www.debian.org/)
-5. [Zorin OS](https://zorin.com/)
-6. [Kali Linux](https://www.kali.org/)
-7. Other Ubuntu flavours like [Kubuntu](https://kubuntu.org/), [Ubuntu unity](https://ubuntuunity.org/) etc.
+* [Linux Mint](https://linuxmint.com/) (Ubuntu-based | Debian-based if using LMDE (Linux Mint Debian Edition))
+* [Pop!_OS](https://pop.system76.com/) (Ubuntu-based)
+* [elementary OS](https://elementary.io/) (Ubuntu-based)
+* [Debian](https://www.debian.org/) (Debian)
+* [Zorin OS](https://zorin.com/) (Ubuntu-based)
+* [Kali Linux](https://www.kali.org/) (Debian-based)
+* Other Ubuntu flavours like [Kubuntu](https://kubuntu.org/), [Ubuntu Unity](https://ubuntuunity.org/) etc. (Ubuntu-based)
 
-# Hardware Requirements
+# Required Hardware
 
-* USB-C to USB adapter
-* Wired internet connection (Ethernet/USB tethering) or Wi-Fi adapter compatible with Linux
+* USB-C to USB-A adapter
+* Internet connection - either **wired** (using an USB Ethernet dongle or USB tethering from a phone) or **wireless** (using a Linux-compatible Wi-Fi adapter)
```

Comment:
I think it would be better to mention that the internal Wi-Fi works with some extra steps. You could link to the wifi guide. Right now users might think that they have to get an external dongle for Wi-Fi.

Reply:
It's a guide for the official ISOs, so no, wifi guide won't fit here.

Reply:
I forgot that debian doesn't have a patched ISO

Reply:
I made it more clear to the user that the internal Wi-Fi will be available post-install with some extra setup.